ui/push-health/ClassificationGroup.jsx

```diff
@@ -80,12 +81,14 @@ class ClassificationGroup extends React.PureComponent {
             className="pointable w-100"
             onClick={this.toggleDetails}
             color={headerColor}
+            role="button"
+            aria-expanded={detailsShowing}
           >
             {name} : {Object.keys(group).length}
             <FontAwesomeIcon
               icon={expandIcon}
               className="ml-1"
-              title="expand"
+              title={expandTitle}
```

Comment:
I was thinking this should be an ``aria-label`` rather than a ``title``.  But if it's a ``title``, and the text is appropriate, then I think that would still work for screen readers.

Reply:
Using title adds `aria-labelledby` under the hood: https://fontawesome.com/how-to-use/on-the-web/other-topics/accessibility#svg-semantic

@yogmel This is something that'd be good to add to our docs, it's easy to forget things like this.

Reply:
@camd I have inserted `aria-label` and kept `title`, because the SR was not reading the `aria-label`, so I thought it was safer to keep them both.

Reply:
@sarah-clements Alright! I have edited your card in the Project with this link as a reminder.
I would need a little assistance once I get to that task, so I will let you know.